src/buildExec.ts

```diff
@@ -45,12 +45,12 @@ const isPullRequestFromFork = (): boolean => {
   return (baseLabel.split(':')[0] !== headLabel.split(':')[0]);
 };
 
-const getToken = async (): Promise<string> => {
+const getToken = async (): Promise<string | null> => {
```

Comment:
I think a comment on what to expect from `null` is needed here. Also, you are returning a Promise to null here but further down you are returning `null` as is